datatables/config.yml

```diff
@@ -0,0 +1,50 @@
+---
+title: Creating a Data Table with CSV
+description: Overview of creating a Data Table from a CSV file
+categories:
+- Architect
+steps:
+- title: Introduction
+  content: |
+    Data tables allow organizations to store data locally and ensure that Architect can access it within an interaction. They can be used to access data sets larger than allowed by a switch statement. 
```

Comment:
Hey Prince.  Looks good.  Just a couple of pieces of feedback.

1.  You might want to add a brief example of maybe some of the types of data that can be stored with data tables.  Do you have any use cases you might see coming up over and over that a reader coming to the tutorial might be searching to solve so that when they look at data tables, they go "Yes!! This is exactly why I want to use this."

2.  Are there things you DONT want to store in data tables that you want to communicate.  I have seen customers try to do some pretty outlandish things with our product and if you have any feedback in the tutorial that might say something "DONT DO X" it can save developers a pretty significant amount of time.